RecoMET/METPUSubtraction/python/deepMETProducer_cff.py

```diff
@@ -1,8 +1,49 @@
 import FWCore.ParameterSet.Config as cms
 
-from RecoMET.METPUSubtraction.deepMETProducer_cfi import deepMETProducer
+from RecoMET.METPUSubtraction.deepMETProducer_cfi import deepMETProducer as _deepMETProducer
+
+deepMETsResolutionTune = _deepMETProducer.clone()
+deepMETsResponseTune = _deepMETProducer.clone(
+    graph_path = 'RecoMET/METPUSubtraction/data/models/deepmet/deepmet_resp_v1_2018/model.graphdef',
+)
+
+from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+phase2_common.toModify(
+    deepMETsResolutionTune,
+    max_n_pf=12500,
+    graph_path="RecoMET/METPUSubtraction/data/models/deepmet/deepmet_v1_phase2/model.graphdef"
+)
+phase2_common.toModify(
+    deepMETsResponseTune,
+    max_n_pf=12500,
+    graph_path="RecoMET/METPUSubtraction/data/models/deepmet/deepmet_resp_v1_phase2/model.graphdef"
+)
+
+from Configuration.Eras.Modifier_run2_jme_2016_cff import run2_jme_2016
+run2_jme_2016.toModify(
+    deepMETsResponseTune,
+    graph_path="RecoMET/METPUSubtraction/data/models/deepmet/deepmet_resp_v1_2016/model.graphdef"
+)
+
 from RecoMET.METPUSubtraction.deepMETSonicProducer_cff import deepMETSonicProducer as _deepMETSonicProducer
 from Configuration.ProcessModifiers.deepMETSonicTriton_cff import deepMETSonicTriton
 
-deepMETSonicTriton.toReplaceWith(deepMETProducer,_deepMETSonicProducer)
+# propagate possible parameter updates
+_deepMETsResolutionTuneSonic = _deepMETSonicProducer.clone(
+    max_n_pf = deepMETsResolutionTune.max_n_pf,
+    Client = dict(
+        #modelVersion = deepMETsResolutionTune.graph_path.value().split('/')[-2],
+        modelVersion = "1"
```

Comment:
this is inconsistent with the above changes in `graph_path`, which assume all model files are moved to `[version_name]/model.graphdef`.

Reply:
Right, but when I went with the commented out version (which I forgot to remove), I got the error
`----- Begin Fatal Exception 20-May-2022 15:19:05 CDT-----------------------
An exception of category 'TritonFailure' occurred while
   [0] Calling beginStream for module DeepMETSonicProducer/'deepMETsResolutionTune'
Exception Message:
TritonClient(): unable to get model config: failed to get model version from specified version string 'deepmet_v1_2018' (details: stol), version should be an integral value > 0
----- End Fatal Exception -------------------------------------------------`
I think Triton requires the model version have an integer as the name

Reply:
Ah, that is unfortunate. I thought when they had changed the model version to a string that they would accept arbitrary values, but apparently not. https://github.com/triton-inference-server/server/blob/fd17daff8ed191660c7877316304cae2fd1084aa/docs/model_repository.md#model-versions says:
> The subdirectories that are not numerically named, or have names that start with zero (0) will be ignored.

Reply:
Yeah, it kind of puts us in a weird situation because directories like '1' and '2' are so non-descriptive.  I would be happy to move the right model files to live only in those directories, but then I think everyone else wouldn't know what they are

Reply:
Indeed, as noted in https://github.com/cms-data/RecoMET-METPUSubtraction/pull/6, needs some more discussion.